tikv-configuration-file.md

```diff
@@ -383,7 +383,7 @@ raftstore 相关的配置项。
 + 默认值：3s
 + 最小值：0
 
-### `raftstore.hibernate-regions` (**实验特性**)
+### `hibernate-regions` (**实验特性**)
```

Comment:
How about putting the config items in alphabetical order?

Reply:
Good idea. I'll do that in a separate PR.